doc/libpmem2/pmem2_config_get_alignment.3.md

```diff
@@ -72,6 +72,9 @@ On all systems:
 * **PMEM2\_E\_FILE\_HANDLE\_NOT\_SET** - config doesn't contain the file handle
 (see **pmem2_config_set_fd**(3), **pmem2_config_set_handle**(3)).
 
+* **PMEM2\_E\_INVALID\_ALIGNMENT\_VALUE** - alignment value is zero,
+system cannot detect proper alignment value.
```

Comment:
operating system returned unexpected alignment value (eg. it is not a power of two)